owners/src/owners_bot.js

```diff
@@ -125,6 +129,34 @@ class OwnersBot {
     await this.runOwnersCheck(github, pr);
   }
 
+  /**
+   * Adds a comment tagging always-notify owners of changed files.
+   *
+   * @param {!GitHub} github GitHub API interface.
+   * @param {!number} prNumber pull request number.
+   * @param {!FileTreeMap} fileTreeMap map from filenames to ownership subtrees.
+   */
+  async createNotifications(github, prNumber, fileTreeMap) {
+    const botComments = await github.getBotComments(prNumber);
```

Comment:
Do you want the owners bot to leave 1 comment overall or multiple?

Reply:
For now one, but leaving open the option of there being multiple